client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -0,0 +1,390 @@
+	},
+} );
+
+const ConfirmationModalList = styled.ul( {
+	marginBottom: '16px',
+} );
+const ConfirmationModalInputTitle = styled.p( {
+	marginBottom: '8px',
+} );
+
+const ConfirmationModalInput = styled( FormInput )( {
+	marginBottom: '26px!important',
+} );
+
+const ConfirmationModalInputSiteSlug = styled.span( {
+	color: '#D63638',
+} );
+
+const ConfirmationModalContainer = styled.div( {
+	display: 'flex',
+	flexDirection: 'row',
+	'@media screen and (max-width: 768px)': {
+		flexDirection: 'column',
+		'.button': { flexGrow: 1 },
+	},
+} );
+
+const StagingSyncCardFooter = styled.p( {
+	fontSize: '14px',
+	fontStyle: 'italic',
+	color: '#646970',
+	marginTop: '16px',
+	lineHeight: '21px',
+	marginBottom: '0px',
+	'& a': {
+		color: '#646970',
+		textDecoration: 'underline',
+	},
+} );
+
+const SyncContainerTitle = styled.p( {
+	fontWeight: 500,
+	marginTop: '0px',
+	marginBottom: '0px',
+} );
+
+const SyncContainerContent = styled.p( {
+	fontWeight: 400,
+	lineHeight: '24px',
+	marginTop: '16px',
+	marginBottom: '16px',
+} );
+
+const FormSelectContainer = styled.div( {
+	marginBottom: '16px',
+	marginTop: '8px',
+	'@media screen and (max-width: 768px)': {
+		display: 'flex',
+		flexDirection: 'column',
+	},
+} );
+
+const OptionsTreeTitle = styled.p( {
+	fontWeight: 500,
+	marginTop: '0px',
+	marginBottom: '16px',
+} );
+
+interface StagingCardProps {
+	onPull: () => void;
+	onPush: ( items?: string[] ) => void;
+	disabled: boolean;
+	siteSlug: string;
+}
+
+interface ProductionCardProps {
+	onPull: ( items?: string[] ) => void;
+	onPush: () => void;
+	disabled: boolean;
+}
+
+const StagingToProductionSync = ( {
+	disabled,
+	siteSlug,
+	onSelectItems,
+	selectedItems,
+	isSyncButtonDisabled,
+	onConfirm,
+}: {
+	disabled: boolean;
+	siteSlug: string;
+	onSelectItems: ( items: CheckboxOptionItem[] ) => void;
+	selectedItems: string[];
+	isSyncButtonDisabled: boolean;
+	onConfirm: () => void;
+} ) => {
+	const [ typedSiteName, setTypedSiteName ] = useState( '' );
+	const translate = useTranslate();
+	return (
+		<>
+			<OptionsTreeTitle>{ translate( 'Synchronize the following:' ) }</OptionsTreeTitle>
+			<SyncOptionsPanel
+				items={ synchronizationOptions }
+				disabled={ disabled }
+				onChange={ onSelectItems }
+			></SyncOptionsPanel>
+			<ConfirmationModalContainer>
+				<ConfirmationModal
+					disabled={ disabled || isSyncButtonDisabled }
+					isConfirmationDisabled={ typedSiteName !== siteSlug }
+					isPrimary={ true }
+					onConfirm={ onConfirm }
+					modalTitle={ translate( 'You’re about to update your production site' ) }
+					extraModalContent={
+						<div>
+							<p>
+								{ translate(
+									'Synchronizing your production site will overwrite the following items with their equivalents from the staging site:'
+								) }
+							</p>
+							<ConfirmationModalList>
+								{ selectedItems.map( ( item ) => {
+									return <li key={ item }>{ item }</li>;
+								} ) }
+							</ConfirmationModalList>
```

Comment:
Should we re-iterate the backup message here? "We'll automatically back up your site..."

Reply:
Reminding users and making them feel safe about it makes sense. On the other side, the message would be visible at the same time in both the popup and on the page inline. What do you think?

Reply:
I don't have a strong opinion about it. @javierarce ?

Reply:
I prefer only to show it once. Right now, it's pretty noticeable next to the main CTA. The reason for not adding it in the modal is that it's preferable for the user to focus only on the task at hand and have fewer things to pay attention to.